src/features/home/components/loginBottomSheet/LoginBottomSheet.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+
+import BandiBoodi3DImage from '@/assets/images/bandiboodi-3d.png';
+import LogoImage from '@/assets/images/logo.png';
+import { BottomSheet, LoginIconSet, ToolTip, Typography } from '@/components';
+
+import { StarBg } from './StarBg';
+
+interface LoginBottomSheetProps {
+  open: boolean;
+  onClose: VoidFunction;
+}
+
+export const LoginBottomSheet = ({ open, onClose }: LoginBottomSheetProps) => {
+  return (
+    <BottomSheet open={open} onDismiss={onClose} fixedMaxHeight={504}>
+      <div className="relative w-full h-[490px] flex flex-col items-center pt-sm">
+        <StarBg />
+        <div className="flex flex-col gap-4xs px-[49px] items-center">
+          <Image src={LogoImage} width={133} alt="logo" />
+          <Typography className="text-blue-50 font-insungit font-medium">내가 직접 그리는 나의 인생 지도</Typography>
```

Comment:
거의 동일한 게 Typography type에 있네~ 둘다 로그인 화면에 사용되는 부분이라 통일해도 좋을 것 같은데~ 일단 나중에 작업할 때 비슷한 거 있다고 참고만 해줘~~~~~


```suggestion
          <Typography type="header1">내가 직접 그리는 나의 인생 지도</Typography>
```

변경 전:
![스크린샷 2024-01-11 203203](https://github.com/depromeet/amazing3-fe/assets/34956359/a079b332-dbbd-4fbc-91b6-08224bee791f)



변경 후:
![image](https://github.com/depromeet/amazing3-fe/assets/34956359/f976665f-d8fb-452d-a81c-e2ff9df916fe)